changelog.txt

```diff
@@ -7,6 +7,7 @@ V.Next
 - [PATCH] Fix accidental code change that disabled PoP for auth code grant flow (#1661)
 - [MINOR] Add flighting parameters to commmandParameters (#1562)
 - [MINOR] Add ropc command and ropc flow to BaseController (#1539)
+- [PATCH] Ensure consistent logging tags
```

Comment:
add the PR number as well
and lets add it to the top

Reply:
Here: da1e62d1ddacd758eb5d0a6cedd9589521475ab8